airflow-core/src/airflow/api_fastapi/core_api/datamodels/plugins.py

```diff
@@ -39,6 +39,15 @@ class FastAPIAppResponse(BaseModel):
     name: str
 
 
+class FastAPIRootMiddlewareResponse(BaseModel):
+    """Serializer for Plugin FastAPI root middleware responses."""
+
+    model_config = ConfigDict(extra="allow")
+
+    middleware: str
+    name: str
```

Comment:
```suggestion
    middleware: str
    name: str
    
    args: list[Any] | None = None
    kwargs: dict[str, Any] | None = None
```

feels better than allow extra

Reply:
Oh, is this used just for the API response? In which case we probably don't want to include args and kwargs at all, as that might be sensitive, so

```suggestion
    middleware: str
    name: str
```

Reply:
args and kwargs are removed. Other extra fields are still returned. (That follow the same pattern as what we do for other Plugin fields, I think we should do the same.)